.github/workflows/airflow-operator.yml

```diff
@@ -0,0 +1,65 @@
+name: Run Python Armada Airflow Operator Linting and Unit Tests
+
+on:
+  push:
+    branches-ignore:
+      - master
+    paths:
+      - 'client/python/**'
+      - 'build/python-client/**'
+      - 'pkg/api/*.proto'
+      - '.github/workflows/python.yml'
+      - 'docs/python_armada_client.md'
+      - 'scripts/build-python-client.sh'
+      - 'third_party/airflow/**'
+      - 'build/airflow-operator/**'
+      - 'pkg/api/jobservice/*.proto'
+      - '.github/workflows/airflow-operator.yml'
+      - 'docs/python_airflow_operator.md'
+      - 'scripts/build-airflow-operator.sh'
+      - 'makefile'
+
+  pull_request:
+    branches-ignore:
+      - gh-pages
+    paths:
+      - 'client/python/**'
+      - 'build/python-client/**'
+      - 'pkg/api/*.proto'
+      - '.github/workflows/python.yml'
+      - 'docs/python_armada_client.md'
+      - 'scripts/build-python-client.sh'
+      - 'third_party/airflow/**'
+      - 'build/airflow-operator/**'
+      - 'pkg/api/jobservice/*.proto'
+      - '.github/workflows/airflow-operator.yml'
+      - 'docs/python_airflow_operator.md'
+      - 'scripts/build-airflow-operator.sh'
+      - 'makefile'
+
+jobs:
+  run-tox:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - run: pip install tox poetry
+      # Generate the proto files for python, required for later steps
+      - run: make python
+      - run: |
+          poetry install -v
+          poetry build
+        working-directory: client/python
```

Comment:
I'm still confused then, because if it's installed into a poetry-specific venv, why does installing it make a difference the later tox runs (it's structured such that it's a dependency for them)?

Reply:
Talked to @kannon92 about this in slack:
- We need to create a wheel file for the python armada client for the airflow operator to consume.
- Tomorrow, we'll see if we can get that wheel by just running `poetry build` (without `poetry install`).
- Also a comment should be added about why we're doing this build here.

This makes a lot more sense now. Thanks for the chat in slack to give me understanding.